src/app/system/external-services/amazon-s3/amazon-s3.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container mat-elevation-z8">
+
+  <table mat-table [dataSource]="dataSource" matSort>
+
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+      <td mat-cell *matCellDef="let configuration"> {{ configuration.name }}&nbsp;&nbsp;
```

Comment:
```suggestion
      <td mat-cell *matCellDef="let configuration"> {{ configuration.name }} </td>
```
Update for all cases in all files.